tools/validators/ontology_validator/yamlformat/validator/presubmit_validate_types_lib.py

```diff
@@ -595,6 +595,30 @@ def _PrintType(ns, et):
     print('  HAS REDUNDANT LOCAL FIELDS')
 
 
+def _printFindings(findings, filter_text):
+  """Prints the findings of the ontology validation
+
+  Args:
+    findings: a list of Finding objects.
+    filter_text: command line arguments. The only available argument is
+      match:<value>' which will simply perform a simple string 'contains' on
```

Comment:
Possible nit? Is there supposed to be an apostrophe after `match:<value>`? I feel like that perhaps implies regex but that's not being used here

Reply:
good catch there is a missing `